main.rb

```diff
@@ -0,0 +1,152 @@
+# test.rb
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+require './book'
+# rubocop:disable all
+# Create class App
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+
+    person_type = gets.chomp
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
+      return
+    end
+
+    print 'Age: '
+
+    age = gets.chomp
+
+    print 'Name: '
+
+    name = gets.chomp
+
+    person =
+      case person_type
+      when '1'
+        print 'Has parent permission? [Y/N]: '
+        # *If needed, ask for parameters for the option.
+        parent_permission = gets.chomp
+        parent_permission = parent_permission.downcase == 'y'
+
+        Student.new(age, name, parent_permission)
+      when '2'
+        print 'Specialization: '
+
+        specialization = gets.chomp
+
+        Teacher.new(age, specialization, name)
+      end
+
+    @people << person
+    puts 'Person created'
+  end
+
+  # *Create a book.
+  def create_book
+    print 'Title: '
+
+    title = gets.chomp
+
+    print 'Author: '
+
+    author = gets.chomp
+
+    @books << Book.new(title, author)
+    puts 'Book created'
+  end
+
+  # *Create a rental.
+  def create_rental
+    puts 'Select a book from the following list by number'
+    @books.each_with_index do |book, index|
+      puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}"
+    end
+
+    book_index = gets.chomp.to_i
+
+    puts "\nSelect a person from the following list by number (not id)"
+    @people.each_with_index do |person, index|
+      puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+
+    person_index = gets.chomp.to_i
+
+    print "\nDate: "
+
+    date = gets.chomp
+
+    @rentals << Rental.new(date, @books[book_index], @people[person_index])
```

Comment:
- [ ] Im getting an error when I tried to create a rental, please take a look at this and fix it. 🚀 
    
    ![imagen](https://user-images.githubusercontent.com/60789317/141517949-00d217e0-9109-4dd3-809b-c92bd4615957.png)

Reply:
- [ ] I wasn't able to test the rental list due to the created rental error, please fix the `create_rental` method. 😺

Reply:
This issue has been addressed well. Great job! :pray: :pray: